dpanonymize/__init__.py

```diff
@@ -56,7 +58,7 @@ def anonymize(self, **kwargs) -> None:
             module.remove_pii(self.file_path, self.general_path)
 
     def __repr__(self):
-        return f"<{self.file_path.name}>"
+        return f"<{self.file_path.name} {self.dtype}"
```

Comment:
Why is the `self.dtype` necessary?

Reply:
Just to print and see if the functions are working well detecting the correct data type for each file path. Mostly for myself.